### ISSUE TITLE
Crash XYIntUtils kotlin 1.4.30

### ISSUE THREAD

**Issue by droidmatic34** [cashapp/contour#120]
```
.FooterView$4.invoke(FooterView.kt:47)
        at ....FooterView$4.invoke(FooterView.kt:13)
        at com.squareup.contour.utils.XYIntUtilsKt$unwrapYIntLambda$1.invoke(XYIntUtils.kt:51)
        at com.squareup.contour.utils.XYIntUtilsKt$unwrapYIntLambda$1.invoke(Unknown Source:2)
        at com.squareup.contour.constraints.Constraint.resolve(Constraint.kt:48)
        at com.squareup.contour.solvers.SimpleAxisSolver.measureSpec(SimpleAxisSolver.kt:173)
        at com.squareup.contour.ContourLayout$LayoutSpec.measureSelf$contour_release(ContourLayout.kt:758)
        at com.squareup.contour.solvers.SimpleAxisSolver.max(SimpleAxisSolver.kt:110)
        at com.squareup.contour.ContourLayout$LayoutSpec.bottom-h0YXg9w$contour_release(ContourLayout.kt:729)
        at com.squareup.contour.ContourLayout.bottom-h0YXg9w(ContourLayout.kt:469)
        at ....FooterView$6.invoke(FooterView.kt:52)
        at ....FooterView$6.invoke(FooterView.kt:13)
        at com.squareup.contour.utils.XYIntUtilsKt$unwrapYIntLambda$1.invoke(XYIntUtils.kt:51)
        at com.squareup.contour.utils.XYIntUtilsKt$unwrapYIntLambda$1.invoke(Unknown Source:2)
        at com.squareup.contour.constraints.Constraint.resolve(Constraint.kt:48)
        at com.squareup.contour.solvers.SimpleAxisSolver.resolveAxis(SimpleAxisSolver.kt:131)
        at com.squareup.contour.solvers.SimpleAxisSolver.max(SimpleAxisSolver.kt:111)
        at com.squareup.contour.ContourLayout$LayoutSpec.bottom-h0YXg9w$contour_release(ContourLayout.kt:729)
        at com.squareup.contour.ContourLayout.bottom-h0YXg9w(ContourLayout.kt:469)
        at ....FooterView$11.invoke(FooterView.kt:62)
        at ....FooterView$11.invoke(FooterView.kt:13)
        at com.squareup.contour.utils.XYIntUtilsKt$unwrapYIntLambda$1.invoke(XYIntUtils.kt:51)
        at com.squareup.contour.utils.XYIntUtilsKt$unwrapYIntLambda$1.invoke(Unknown Source:2)
        at com.squareup.contour.constraints.Constraint.resolve(Constraint.kt:48)
        at com.squareup.contour.solvers.SimpleAxisSolver.resolveAxis(SimpleAxisSolver.kt:131)
        at com.squareup.contour.solvers.SimpleAxisSolver.max(SimpleAxisSolver.kt:111)
        at com.squareup.contour.ContourLayout$LayoutSpec.bottom-h0YXg9w$contour_release(ContourLayout.kt:729)
        at com.squareup.contour.ContourLayout.bottom-h0YXg9w(ContourLayout.kt:469)
        at ....FooterView$16.invoke(FooterView.kt:75)
        at ....FooterView$16.invoke(FooterView.kt:13)
        at com.squareup.contour.utils.XYIntUtilsKt$unwrapYIntToYIntLambda$1.invoke(XYIntUtils.kt:61)
        at com.squareup.contour.utils.XYIntUtilsKt$unwrapYIntToYIntLambda$1.invoke(Unknown Source:6)
        at com.squareup.contour.constraints.SizeConfig.resolve(SizeConfig.kt:29)
        at com.squareup.contour.ContourLayout.onMeasure(ContourLayout.kt:213)
        at android.view.View.measure(View.java:25466)
```

**Comment by saket** [comment]
Should be fixed by https://github.com/cashapp/contour/commit/adb8ad3b7539d9eb531c4ff7132fda18375b20dd. You can try it out using `1.1.0-SNAPSHOT` until it's released.

**Comment by droidmatic34** [comment]
fixed in 1.1.0-snapshot.

Thanks
